userd/ui/ui.go

```diff
@@ -56,15 +56,15 @@ var hasKDialogExecutable = func() bool {
 
 func MockHasZenityExecutable(f func() bool) func() {
 	oldHasZenityExecutable := hasZenityExecutable
-	hasZenityExecutable := f
+	hasZenityExecutable = f
 	return func() {
 		hasZenityExecutable = oldHasZenityExecutable
 	}
 }
 
 func MockHasKDialogExecutable(f func() bool) func() {
 	oldHasKDialogExecutable := hasKDialogExecutable
-	hasKDialogExecutable := f
```

Comment:
Nice catch. Do we need this for 2.32?

Reply:
ouch /o\. No, it's just affecting the tests.